versions/3.2.0.md

```diff
@@ -99,6 +99,19 @@ Some examples of possible media type definitions:
 The HTTP Status Codes are used to indicate the status of the executed operation. 
 The available status codes are defined by [RFC7231](https://tools.ietf.org/html/rfc7231#section-6) and registered status codes are listed in the [IANA Status Code Registry](https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml).
 
+##### <a name="undefinedAndImplementationDefinedBehavior"></a>Undefined and Implementation-Defined Behavior
+
+This specification deems certain situations to have either _undefined_ or _implementation-defined_ behavior.
+
+Behavior described as _undefined_ is likely, at least in some circumstances, to contradict the specification.
```

Comment:
```suggestion
Behavior described as _undefined_ is likely, at least in some circumstances, to result in outcomes that contradict the specification.
```
This change seems to have got lost.

Reply:
Oops I spliced a commit off the branch again when porting.  I've pushed a fix now.